blog/_posts/2025-05-29-azure-monitor-grafana-dashboards-portal.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Azure Monitor dashboards with Grafana in Azure Portal"
+description: "Explore the new Grafana dashboards built directly into Azure Monitor and learn how they empower observability and when to reach for Azure Managed Grafana."
+date: 2025-09-18
+authors: 
+   - Aritra Ghosh
+   - Kayode Prince
+categories: 
+- operations
+tags:
+  - azure-monitor
+  - metrics
+  - grafana
+  - observability 
+  - aks
+---
+
+
+### Introduction
+
+As Kubernetes adoption accelerates, engineers need streamlined, cost-effective tools for cluster observability. Until now, this often meant deploying and managing separate monitoring stacks. Azure Monitor's latest integration with Grafana changes this: cluster insights are now just a click away in the Azure portal.
+
+We are thrilled to announce that Azure Kubernetes Service (AKS) now offers native Grafana dashboards within the Azure portal at no additional cost. This integration eliminates the complexity of maintaining separate visualization tools while delivering Grafana's powerful capabilities directly where you need them. Metrics from Container Insights, the Kubernetes metrics server, and any configured Azure Managed Prometheus endpoints are available out-of-the-box, providing comprehensive cluster observability.
+
```

Comment:
can we add doc links to Azure Managed Prometheus, Container Insights, Azure Managed Grafana on their first references in the doc?